packages/ckeditor5-widget/tests/widgetresize.js

```diff
@@ -156,13 +156,27 @@ describe( 'WidgetResize', () => {
 		} );
 
 		it( 'it\'s visible once the widget is focused', () => {
-			// Widget is focused by default.
+			const widgetResizePlugin = editor.plugins.get( WidgetResize );
+			widgetResizePlugin.select( widgetResizePlugin.selectedResizer );
+
 			const allResizers = editor.ui.getEditableElement().querySelectorAll( '.ck-widget__resizer__handle' );
```

Comment:
We were supposed to change these tests, so that we don't fetch all resizer through DOM selectors. But now I wonder what is the purpose at all of this test (and maybe similar tests).

1.  For sure we need tests in `Resizer` class that will check if:
    1.  `isVisible` is properly toggled when `isEnabled` and `isSelected` changes,
    2.  changing `isVisible` makes a change in view (adding a CSS class).
2.  However, I think that `WidgetResize` can be limited to just:
    1.  simple tests for `select()` and `deselect()` methods (just check if the resizer is selected and if `selectedResizer` is set),
    2.  whether `select()` and `deselect()` is used on selection changes, so the tests should be named like: "selects resizer when related view element becomes selected".

I am not sure if we need any other tests but we should have these tests.

Please make the test suites a little more appropriate to what the classes are responsible for.